src/citrine/_serialization/properties.py

```diff
@@ -655,7 +655,7 @@ def _deserialize(self, data: dict) -> typing.Any:
             raise AttributeError("Tried to deserialize to {!r}, which has no fields and is not an"
                                  " explicitly serializable class".format(self.klass))
 
-        instance = self.klass.__new__(self.klass, {})
+        instance = self.klass.__new__(self.klass)
```

Comment:
This is the piece that has required us on many occasions to need to define a constructor for an object which will never be constructed by the user. In the context of this PR, it would require me to add one to the [AsyncDefaultPredictor](https://github.com/CitrineInformatics/citrine-python/blob/58d82b829dad95c6a72703b6648da107d2d4101e/src/citrine/resources/predictor.py#L22) class.